lkmpg.tex

```diff
@@ -2044,6 +2044,60 @@ \subsection{Likely and Unlikely conditions}
 That avoids flushing the processor pipeline.
 The opposite happens if you use the \cpp|likely| macro.
 
+\subsection{Static keys}
+\label{sec:static_key}
```

Comment:
To consist of the subsection title, please add `s` here.

Reply:
OK